misc/prototypes.h

```diff
@@ -374,6 +380,16 @@ struct hostent *gethostbyname(const char *name);
 struct hostent *gethostbyaddr(const void *addr, socklen_t len, enum uft_socket_domain type);
 int getaddrinfo(const char *node, const char *service, const struct addrinfo *hints, struct addrinfo **res);
 void freeaddrinfo(struct addrinfo *res);
+
+#include <netinet/in.h>
+#include <arpa/inet.h>
+int inet_pton(enum uft_socket_domain af, const char *src, void *dst);
+const char *inet_ntop(enum uft_socket_domain af, const void *src, char *dst, socklen_t size);
+int inet_aton(const char *cp, struct in_addr *inp);
+
+in_addr_t inet_addr(const char *cp);
+in_addr_t inet_network(const char *cp);
+char *inet_ntoa(struct in_addr in);
```

Comment:
Isn't it better moving `inet_ntoa()` under the `inet_aton()`?

Reply:
Okay. I just reordered as you said.